src/cli.ts

```diff
@@ -50,10 +51,20 @@ const logToolbeltVersion = () => {
   log.debug(`Toolbelt version: ${pkg.version}`)
 }
 
+const validToken = () => {
+  const token = getToken()
+  if (!token) { return false }
+
+  const decoded = decode(token, {complete: true})
```

Comment:
Can you be sure the token always returns decoded? what if `getToken()` returns an empty string, or a malformed token?

Reply:
(because in the next line you're trying to access `decoded.payload.exp`)

jsonwebtoken.decode has this signature:
```ts
decode(token: string, options?: DecodeOptions): string | { [key: string]: any; }
```